.gitignore

```diff
@@ -27,3 +27,8 @@ bundle/
 .vagrant/
 phantomjsdriver.log
 selenium-debug.log
+src/sandstorm/*.capnp.c++
+src/sandstorm/*.capnp.h
```

Comment:
Yes, can you please remove these? They shouldn't be needed and could cause confusion in the future.